README.md

```diff
@@ -4,68 +4,84 @@ Healthcheck url, must be on city network: https://citybase.austinmobility.io/
 
 ## Description
 
-Flask based endpoint used to update Knack transaction records based on events delivered by Citybase.
+Flask based endpoint ("the postback") used to update Knack transaction records based on events delivered by Citybase.
 
 ## Integration with Knack
 
-Citybase is the payment provider for the Over the Street and Lamppost Banner invoices.
-Once a reservation is created and approved in the Knack app, an invoice is generated and marked as ready to pay. Custom [javascript](https://github.com/cityofaustin/atd-knack/blob/master/code/street-banner/street-banner.js#L417) creates a payload to send to the [Citybase endpoint](https://invoice-service.prod.cityba.se/invoices/austin_tx_transportation/street_banner). This request responds with a url to process the payment request. Note, the amount must be in pennies.
+Citybase is the payment provider for two Knack apps, Street Banners and Neighborhood Block Parties in Smart Mobility.
+Once a reservation is created and approved in the Knack app, an invoice is generated and marked as ready to pay. Custom javascript ([example](https://github.com/cityofaustin/atd-knack/blob/master/code/street-banner/street-banner.js#L417)) creates a payload to send to the [Citybase endpoint](https://invoice-service.prod.cityba.se/invoices/austin_tx_transportation/street_banner). This request responds with a url to process the payment request. Note, the amount must be in pennies.
 
-Once a user has completed the payment transaction, Citybase sends a postback and waits for us to confirm receipt. Citybase does not consider the payment complete until it receives a 200 response to the request.
+Once a user has completed the payment transaction, Citybase sends a payload and waits for a 200 reply.
```

Comment:
I think it's in the name, postback, but would it be helpful to say explicitly what HTTP verb is being used by citybase, presumably `POST`?